### DISCUSSION TITLE
jump to position

### DISCUSSION THREAD

**Discussion by anon-gib** [PaulWoitaschek/Voice#2452] [Ideas]
I remember this being an option once. Some books have no chapters and can be quite long. Using the seek bar with fidelity is quite difficult on a 22 hour long book, with no chapters.

Please add a "jump to position" option so we can manually type in a timestamp or similar. 

Thanks! Love the app! (Using 8.1.3 from the play store)

**Comment by mystyc** [comment]
Yeah, I remember this as well.  For awhile, I thought I was misremembering things.
One day, I wanted to jump to a certain time, but I could not find the setting.  It wasn't there.  It just wasn't there.  Then I wondered if it was ever really there in the first place. Maybe this feature was always just a wish...
Sometimes I'll catch myself looking for it and it's like, "No!  Stop it!  It's not there. Just accept it.  Stop doing this to yourself."
Perhaps, one day, the absence of this feature will feel less palpable.  
Cheers.
